Cloud Gateways: Update troubleshooting section (#7197)
Kong/docs.konghq.com: app/konnect/gateway-manager/troubleshoot.md

content_type: how-to
---

## Upgrading managed data planes

**Problem** 

The data plane version is out of date. 

**Solution**

With Dedicated Cloud Gateways, {{site.base_gateway}} updates can be performed with the {{site.konnect_short_name}} manager or API. {{site.konnect_short_name}} will automatically provision new data plane nodes and slowly migrate traffic while monitoring for errors. For more information, read the [Upgrade documentation](/konnect/gateway-manager/data-plane-nodes/upgrade/).

## DNS capabilities

**Problem**

> APIs need to be availble publicly for external applications and privately for internal traffic. 

Reviewer: ```suggestion
APIs need to be available publicly for external applications and privately for internal traffic. 
```

**Solution**

> Cloud Gateways offer both a **Public** and **Private** mode. In public mode, you can expose your APIs to the internet, 

Reviewer: ```suggestion
Cloud Gateways offer both a **Public** and **Private** mode.
```
Looks like the second half of this sentence is missing, and then the description below covers both - so I think it's better to remove.

> A public or private mode to decide who can view your APIs. In public mode, powered by Kong's public Edge DNS for clusters, you can expose your APIs to the internet. Private mode allows you to create internal APIs that won't be availble on the public internet. 

Reviewer: ```suggestion
You can choose public or private mode to decide who can view your APIs. In public mode, powered by Kong's public Edge DNS for clusters, you can expose your APIs to the internet. In private mode, you can create internal APIs that won't be available on the public internet. 
```
Some rephrasing for flow and parallelism.

## Custom domains

**Problem**

> How do you configure custom domains for managed data planes

Reviewer: ```suggestion
How do you configure custom domains for managed data planes?
```

content_type: how-to
---

> ### Out of sync data plane node

Reviewer: ```suggestion
## Out of sync data plane node
```
We have to keep the semantic order of headers. H3 can't come directly after H1.

If you are unable to resolve sync issues using the above methods, contact
[Kong Support](https://support.konghq.com/).

> ### Missing functionality

Reviewer: ```suggestion
## Missing functionality
```

If your version is up to date but the feature still isn't working, contact
[Kong Support](https://support.konghq.com/).
{% endnavtab %}
{% endnavtabs %}

> ### Kubernetes data plane node installation does not work

Reviewer: ```suggestion
## Kubernetes data plane node installation does not work
```

  --values ./values.yaml
```

> ### Connect a data plane to {{site.konnect_saas}} that is behind a non-trasparent forward proxy.

Reviewer: ```suggestion
## Connect a data plane to {{site.konnect_saas}} that is behind a non-trasparent forward proxy.
```
